media-video/qmplay2/qmplay2-18.04.01.ebuild

```diff
@@ -0,0 +1,111 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PLOCALES="de es fr hu pl ru uk zh"
```

Comment:
Please don't use l10n.eclass. Just install all localizations unconditionally.

Reply:
What reason? l10n.eclass is deprecated?

Reply:
Mostly the logic behind it is deprecated. It might have a few use cases left but this is not one.

Reply:
Can I get more details? Is there any discussion in the mail lists?

Reply:
I think https://gentoo.org/support/news-items/2016-06-23-l10n-use_expand.html is the most relevant doc. LINGUAS flags were broken and were removed. L10N is the new thing but we don't abuse it to remove few tiny files. There's also the policy on not controlling small files via USE flags from few years back but I don't recall where to look for it.

Reply:
Still, they don't require extra dependencies, do not take meaningfully long time and do not take a lot of space.